frontend/src/lib/components/ActivationSidebar/activationLogic.ts

```diff
@@ -375,4 +376,5 @@ export const activationLogic = kea<activationLogicType>([
             }
         },
     })),
+    permanentlyMount(),
```

Comment:
See other comment - basically we don't want to unmount this as it can trigger a lot of other requests